Src/GoogleApis.Auth/OAuth2/ITokenAccess.cs

```diff
@@ -22,19 +22,24 @@
 namespace Google.Apis.Auth.OAuth2
 {
     /// <summary>
-    /// Allows direct control over access token and its retrieval.
+    /// Allows direct retrieval of access tokens to authenticate requests.
     /// This is necessary for workflows where you don't want to use 
     /// <see cref="T:Google.Apis.Services.BaseClientService"/> to access the API.
     /// (e.g. gRPC that implemenents the entire HTTP2 stack internally).
     /// </summary>
     public interface ITokenAccess
```

Comment:
I would be happy to see TokenResponse reminds the same here + we might want to have another interface that inherit from IToeknAccess, IConfigurableHttpClientInitializer and IHttpUnsuccessfulResponseHandler.
Something like ICredential